python/ray/rllib/ppo/ppo.py

```diff
@@ -60,7 +60,7 @@
     # If >1, adds frameskip
     "extra_frameskip": 1,
     # Number of timesteps collected in each outer loop
-    "timesteps_per_batch": 40000,
```

Comment:
needs to be changed back

Reply:
IMO we should keep it at 4k, 40k is very likely to crash any laptop user using this script, as it does each time I run PPO by accident :)

For tuned params, they can refer to the examples.